frontend/workflows/projectSelector/src/project-group.tsx

```diff
@@ -133,6 +139,7 @@ const ProjectGroup: React.FC<ProjectGroupProps> = ({ title, group, displayToggle
   const state = useReducerState();
 
   const [collapsed, setCollapsed] = React.useState(false);
+  const [keyWithQLinksOpen, setKeyWithQLinksOpen] = React.useState("");
```

Comment:
nit: seems like we could push this state down a component level since it's only used in the one parent, just helps to minimize the amount of state to mentally work through when working on this component.